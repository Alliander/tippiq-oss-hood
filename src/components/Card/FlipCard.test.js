import React from 'react';
import renderer from 'react-test-renderer';
import Element from './FlipCard';

test('Renders', () => {
  const component = renderer.create(
    <Element>
      <div>Front</div>
      <div>Back</div>
    </Element>
  );
  const json = component.toJSON();
  expect(json).toMatchSnapshot();
});
