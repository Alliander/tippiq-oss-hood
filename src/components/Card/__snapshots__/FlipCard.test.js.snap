exports[`test Renders 1`] = `
<div
  className="undefined horizontal undefined  undefined"
  onBlur={[Function]}
  onFocus={[Function]}
  tabIndex={0}>
  <div
    className={undefined}>
    <div
      aria-hidden={false}
      className={undefined}
      tabIndex={-1}>
      <div>
        Front
      </div>
    </div>
    <div
      aria-hidden={true}
      className={undefined}
      tabIndex={-1}>
      <div>
        Back
      </div>
    </div>
  </div>
</div>
`;
